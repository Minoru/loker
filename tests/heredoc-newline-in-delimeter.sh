cat <<'hello
there'
hey
hello
there
