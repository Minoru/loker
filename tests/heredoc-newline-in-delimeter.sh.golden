([(First (Pipeline Straight [ComSimple (SimpleCommand [] [Redirection 0 (HereDoc NoStrip 0) [SQuoted "hello\nthere"]] [[Bare "cat"]])]),Seq)],SS {hereDocHandles = [("hello\nthere",0,HereDocQuoted)], hereDocs = fromList [(0,[SQuoted "hey\n"])], numHereDocs = 1})
